.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,39 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
```

Comment:
1) with your real name, no nick names
2) If they try to open a PR and read the template, they already have a commit, so I guess we should say `git commit -s --amend` and force push.

Reply:
Okay, fixed.